package.json

```diff
@@ -46,6 +46,7 @@
     "react-native-blur": "^3.1.1",
     "react-native-interactable": "1.0.0-alpha.1",
     "react-native-swipe-gestures": "^1.0.2",
+    "react-native-text-size": "^2.0.4",
```

Comment:
we need to PR the engine to add this native as soon as possible, cause it takes time till they release versions and the native stuff are has to built there.